### ISSUE TITLE
Fix bug in standard state correction reading

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#268]
@luirink points out this issue:
https://github.com/choderalab/yank/blob/master/Yank/analyze.py#L475

```
                DeltaF_restraints = ncfile.groups['metadata'].variables['standard_state_correction'][0]
```

This may need to be 

```
                DeltaF_restraints = float(ncfile.groups['metadata'].variables['standard_state_correction'][0])
```

**Comment by luirink** [comment]
Also then I get the error of invalid index to scalar variable.:

DeltaF_restraints = float(ncfile.groups['metadata'].variables['standard_state_correction'][0])
File "netCDF4.pyx", line 3028, in netCDF4.Variable.__getitem__ (netCDF4.c:37443)
IndexError: invalid index to scalar variable.

however, I found also this:
https://github.com/Unidata/netcdf4-python/issues/328
can it have to do with this?

both 
DeltaF_restraints = ncfile.groups['metadata'].variables['standard_state_correction'][:]
and
DeltaF_restraints = ncfile.groups['metadata'].variables['standard_state_correction'][()]
at least don't give me errors and the same result for the restraint free energy. I am just not sure if it takes the number it should take..

**Comment by Lnaden** [comment]
This bug should be fixed pre 1.0.
